tungsten_ci_utils/mirror_maven_repo/scrapy-maven.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
+
+import scrapy
+import re
+
+
+class MavenRepositorySpider(scrapy.Spider):
+    name = 'maven-repo-spider'
+    start_urls = ['http://sdnpoc-vrodev.englab.juniper.net/vco-repo']
```

Comment:
Make this configurable (config file or cli parameter)

Reply:
Done.